airbyte-integrations/connectors/source-camunda-history/source_camunda_history/schemas/TODO.md

```diff
@@ -0,0 +1,16 @@
+# TODO: Define your stream schemas
```

Comment:
Remove this file.

Reply:
Camunda is a workflow management system. Only historical events are needed for Data Extraction from ETL perspective. Calling it just Camunda would be misleading since the "history" is only a subset of the API.